### DISCUSSION TITLE
Allow recursive album view?

### DISCUSSION THREAD

**Discussion by AeliusSaionji** [deckerst/aves#1247] [Ideas]
I like aves to always start in my `Camera` photos folder, to keep any downloaded images separated by default. But the Sony camera app likes to create subfolders:

![Imagepipe_16](https://github.com/user-attachments/assets/c6dc00f4-be75-4d0d-8cc2-5c2e4e616234)

![Screenshot_20241016-141301](https://github.com/user-attachments/assets/47c2cacd-da1d-4862-a133-9a43cb5b3f23)

So when browsing `Camera`, certain videos and all Bursts are excluded from view. Moreover, bursts clutter up my albums view, since each burst is detected as a distinct album.

Being able to recursively view a folder would be appeciated. If I could also make bursts not appear as albums, that'd be a nice bonus.

And as usual, thanks for your hard work!

**Comment by AeliusSaionji** [comment]
Alternative: detect bursts as a `Type` alongside svg/hdr/etc, so they can be added to a combined filter view.

Or let users compose a regex filter? Example for the above bursts `\d{8}_\d{9}`

**Comment by deckerst** [comment]
Related: 
- #600 
- #599

As for your need, you could:
1) open Explorer,
2) navigate to the "Camera" folder,
3) tap the path filter at the bottom,
4) select in the Collection page menu `Add shortcut` and/or `Set as home`.

"Path filters" (the kind you get at the bottom in the Explorer) are recursive, contrary to the "Album filters" (the kind you get from the Albums page). The filter chip icon indicates the kind of filter.

**Comment by AeliusSaionji** [comment]
> Path filters" (the kind you get at the bottom in the Explorer) are recursive

Aha! I thought I tried that, but I did not look at the result closely enough. Thanks!
